sentry_sdk/transport.py

```diff
@@ -663,8 +851,12 @@ def make_transport(options):
     # type: (Dict[str, Any]) -> Optional[Transport]
     ref_transport = options["transport"]
 
+    use_http2_transport = options.get("_experiments", {}).get("transport_http2", False)
+
     # By default, we use the http transport class
-    transport_cls = HttpTransport  # type: Type[Transport]
+    transport_cls = (
+        Http2Transport if use_http2_transport else HttpTransport
```

Comment:
Interesting as I would have expected this to blow up during tests. That said since we add `httpcore` as a test dependency, I guess we don't get the issue. I agree, this is dangerous and will make it better.

Reply:
Yeah the tests always have httpcore available.